### PR TITLE
Fix tags so they're consistent across posts

### LINKED ISSUE #1103: Use tags more consistently for Node, .NET, and Vue
Hi folks!

I'm currently updating our Languages and SDKs pages on dev.okta to point to your newest blog posts by way of tags (i.e. https://developer.okta.com/blog/tags/) — see my PR at https://github.com/okta/okta-developer-docs/pull/3124.

Most of the languages seem fine, but there are a few where multiple tags seem to be used for the same thing, meaning that it is impossible to use the tag URLs to get a definitive list of all the posts for a subject in one place. The main problem ones as I see it are:

* Node — you've got node, node-dot-js, and nodejs
* .NET Core — you've got asp-dot-net-core, aspnet-core, aspnetcore, and others
* .NET not Core (Framework?) — you've got asp-dot-net, asp-net, aspnet, dot-net, dotnet, and others
* Vue — you've got vue, vue-dot-js, vue-js, and vuejs

Can you let me know which one I should link to in each case, and then update the tags so that there is one clear tag to use for each of these majot topics? I'll go ahead and choose one for each topic for now so as not to hold up my PR, then I can always change them later if needed.

Thanks!

### PR DESCRIPTION
**Pull request by mraible**
Fixes #1103.

You can see all our current tags at https://developer.okta.com/blog/tags/. 

Changes so far:

- nodejs -> node
- vuejs -> vue

IMHO, I don't think it matters what we use, as long as we're consistent. I chose the tags I'd expect to see on the page and what I'd say when talking to another developer.

### CODE REVIEW

**Review by deepu105 — APPROVED** [review]

**Comment by mraible** [comment]
@deepu105 Should we just merge this as-is and improve tags as we see them going forward? Or do you think we should spend more time refining tags?

**Comment by deepu105** [comment]
I think thats more practical idea. Tomorrow I'll also go through and see if
I can optimise some tags and merge this. Is that ok with you?

**Comment by mraible** [comment]
Works for me!

File: _source/_posts/2017-07-25-oidc-primer-part-2.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ author: micah-silverman
 by: advocate
 communities: [security]
 description: "This blog series is a primer on OIDC. In this post, we'll dive into the mechanics of OIDC and see the various flows in action."
-tags: [oauth, oauth2, oauth2-dot-0, oauth-2-dot-0, openid, openid-connect, oidc]
```
Comment (deepu105) [inline _source/_posts/2017-07-25-oidc-primer-part-2.md:11]: do we need both oauth and oauth2?

Comment (deepu105) [reply _source/_posts/2017-07-25-oidc-primer-part-2.md:11]: also I think we should remove openid, openid-connect and have only oidc

**Review by deepu105 — COMMENTED** [review]
I cleaned up some more tags

**Comment by deepu105** [comment]
@mraible lets merge if this looks good to you

**Pull request merged**